packages/components/addon/components/hds/tabs/index.hbs

```diff
@@ -1,4 +1,78 @@
-{{! ADD YOUR TEMPLATE CONTENT HERE }}
 <div class={{this.classNames}} ...attributes>
-  {{yield}}
-</div>
\ No newline at end of file
+  {{!-- {{yield}} --}}
+
+  {{! Hds-TabList }}
+  <div class="hds-tab-list">
+    <ul class="hds-tab-list__list" role="tablist">
+      {{! Hds-Tab }}
+      <li class="hds-tab" role="presentation">
```

Comment:
```suggestion
      <li class="hds-tab" role="presentation">
```

[suggestion] since this has the role of presentation, I don't think it should get a class name. We can target it by the role.

Reply:
Ok thanks but I'm going to break this out into a child component. I'll see, but I think I would want to keep a class name for the component's root element.